lib/ytnef.c

```diff
@@ -421,6 +421,11 @@ int TNEFFillMapi(TNEFStruct *TNEF, BYTE *data, DWORD size, MAPIProps *p) {
   d = data;
   p->count = SwapDWord((BYTE*)data, 4);
   d += 4;
+  // Arbitrary limit on the amount of properties
+  if (((size_t) p->count) > 1000) {
+     printf("suspecting a corrupt file in MAPI allocation\n");
```

Comment:
Lets prefix this output with "ERROR" so that it's more recognizable and matches the format of other error codes.